docs/embark-anyone.rst

```diff
@@ -0,0 +1,18 @@
+embark-anyone
+=============
+
+.. dfhack-tool::
+    :summary: Allows you to embark as any civilisation, including dead and non-dwarven ones.
+    :tags: embark gameplay armok
+
```

Comment:
Please add an introductory paragraph before the Usage section that at least goes over when to run the command